java/src/jmri/jmrit/beantable/beanedit/SensorDebounceEditAction.java

```diff
@@ -78,14 +78,14 @@ public void actionPerformed(ActionEvent e) {
         basic.addItem(new BeanEditItem(null, null, Bundle.getMessage("SensorDebounceText")));
         basic.addItem(new BeanEditItem(sensorDebounceGlobalCheck, Bundle.getMessage("SensorDebounceUseGlobalText"), null));
         sensorDebounceInactiveSpinner.setModel(
-                new SpinnerNumberModel(Long.valueOf(0L), Long.valueOf(0L), Long.valueOf(1000L), Long.valueOf(1L)));
+                new SpinnerNumberModel(Long.valueOf(0L), Long.valueOf(0L), Long.valueOf(Sensor.MAX_DEBOUNCE), Long.valueOf(1L)));
 //        sensorDebounceInactiveSpinner.setValue(Long.valueOf(0L)); // reset from possible previous use
-        sensorDebounceInactiveSpinner.setPreferredSize(new JTextField(5).getPreferredSize());
+        sensorDebounceInactiveSpinner.setPreferredSize(new JTextField(Long.toString(Sensor.MAX_DEBOUNCE).length()+1).getPreferredSize());
```

Comment:
Would it be better to just let the LayoutManager handle the size of this?  That'll let resize work, etc. I haven't looked at the layout, but usually that leaves lots of size for fields.

Reply:
The class is used in 2 places, layout editor and the bean table grid. If you want to change that, then all the other fields that thing would need to be changed to be consistant.

Reply:
Thanks for thinking about this, but unfortunately, I don't understand your point. java/src/jmri/jmrit/beantable/beanedit/SensorDebounceEditAction.java provides the entire panel for editing this, which is itself included into something else. Why would anything else have to change?